setup.cfg

```diff
@@ -16,7 +16,10 @@ setup_requires = setuptools_scm
 install_requires =
     sunpy>=3
     pyvista
-
+    matplotlib
```

Comment:
I don't think Matplotlib should need to be an installation requirement, do you need it for anything at the moment?

Reply:
Not at the moment, I'll remove the draw_quadrangle method so I'll get rid of this as well.